inspec-iggy.gemspec

```diff
@@ -22,5 +22,5 @@ Gem::Specification.new do |spec|
 
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'inspec', '>=2.2.64', '<3.0.0'
+  spec.add_dependency 'inspec', '>=2.3', '<4.0.0'
```

Comment:
@clintoncwolfe I'm guessing <4.0.0 is pretty safe?

Reply:
2.3 introduced the new plugin API.  No relevant breaking changes on 3.0 for iggy, next pin is 4.